File: ympa_fsharp_functions_server/src/model/UpdateOrderStatusesRequest.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.OrderStateDTO

module UpdateOrderStatusesRequest =

  //#region UpdateOrderStatusesRequest

  [<CLIMutable>]
  type UpdateOrderStatusesRequest = {
    [<JsonProperty(PropertyName = "orders")>]
    Orders : OrderStateDTO[];
  }

  //#endregion

File: src/gatling/scala/org/openapitools/client/model/UpdateOrderStatusesRequest.scala

package org.openapitools.client.model


case class UpdateOrderStatusesRequest (
    /* Список заказов. */
    _orders: List[OrderStateDTO]
)
object UpdateOrderStatusesRequest {
    def toStringBody(var_orders: Object) =
        s"""
        | {
        | "orders":$var_orders
        | }
        """.stripMargin
}
